dialogs.py

```diff
@@ -3,8 +3,13 @@
 from functools import partial
 
 
-def popup(parent, title, details, icon, *, buttons):
+def popup(parent, title, details, icon, *, buttons, theme):
     dialog = tk.Toplevel()
+    window.tk.call("source", "sun-valley.tcl")
+    
+    window.tk.call("set_theme", "%s" % theme)
+    if theme == "light": dialog.attributes("-transparentcolor", "#fafafa")
+    else: dialog.attributes("-transparentcolor", "#1c1c1c")
```

Comment:
```suggestion
    if parent.tk.call("tk", "windowingsystem") == "win32":
        transparent_color = parent.tk.call("ttk::style", "lookup", "TFrame", "-background")
        dialog.wm_attributes("-transparentcolor", transparent_color)
```
I don't know whether this works at all, but should definitely be Windows-only

Reply:
Yes, the `-transparentcolor` option is Windows-only